pkg/controllers/user/logging/utils/embedded.go

```diff
@@ -468,7 +506,17 @@ func newKibanaService(namespace string) *v1.Service {
 	}
 }
 
-func NewESDeployment(namespace string) *v1beta2.Deployment {
+func NewESDeployment(namespace string, obj *v3.ClusterLogging) *v1beta2.Deployment {
```

Comment:
is this function being called from diff packages too? if not, could you lower case the name